docker/generate_dockerfile_cpu.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash 
+docker run --rm repronim/neurodocker:latest generate docker \
+    --pkg-manager apt \
+    --base-image ubuntu:20.04 \
+    --env "DEBIAN_FRONTEND=noninteractive" \
+    --run "chmod 777 /tmp" \
+    --install build-essential git ca-certificates netbase\
+    --miniconda \
+    version="py311_24.4.0-0" \
+    conda_install="python=3.11 gxx_linux-64 notebook numpy=1.26.4 git-annex" \
+    --run "pip install Pillow==9.5.0" \
+    --workdir /voxelwise_tutorials \
+    --run "pip install voxelwise_tutorials" \
+    --run "git clone --depth 1 https://github.com/gallantlab/voxelwise_tutorials.git" \
+    --run "pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu" \
+    --user nonroot \
```

Comment:
Can you try moving the `--user nonroot` flag before line 11 (`pip install Pillow`)? We're still getting some permission errors due to `/voxelwise_tutorials` being owned by root

```
    PermissionError: [Errno 13] Permission denied: '/voxelwise_tutorials/voxelwise_tutorials/tutorials/notebooks/shortclips/.ipynb_checkpoints'
```